src/qml/VariableEditor.qml

```diff
@@ -151,10 +157,15 @@ ColumnLayout {
     text: qsTr("Add a new variable")
 
     onClicked: {
-      table.model.addCustomVariable("new_variable", "");
-      table.positionViewAtIndex(table.count - 1, ListView.visible);
-      // TODO: Use Qt 5.13 itemAtIndex( index )
-      table.children[0].children[table.count].children[0].children[0].forceActiveFocus();
+      let lastAppVariableIndex = 0;
+      for (let i = 0; i < table.count; ++i) {
+        if (table.itemAtIndex(i) && table.itemAtIndex(i).variableType === 0) {
+          lastAppVariableIndex = i;
+        }
+      }
+      table.model.addCustomVariable("new_variable", "", lastAppVariableIndex + 1);
```

Comment:
Can we let go of this last app variable index logic here in favor of renaming the function to `addVariable`, and have the first parameter dictate the scope, i.e. ApplicationScope or ProjectScope (we won't deal with the latter in this PR).

It'd be much, much cleaner.

Reply:
for `addCustomVariable` yes, but we still need to get that lastIndex for these 2 lines: 

```QML
table.positionViewAtIndex(lastAppVariableIndex + 1, ListView.Contain);
table.itemAtIndex(lastAppVariableIndex + 1).forceFocusOnVariableName();
```